app/models/organization.rb

```diff
@@ -48,4 +52,12 @@ def logo_delete
   def delete_logo
     logo.clear if logo_delete == '1'
   end
+
+  def set_github_org
+    if projects.first
+      self.github_org = Project.parse_git_url(projects.first.submitted_github_url)
+                          .split('/')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 20) spaces for indenting an expression spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.